### PR TITLE
New function normalize_audio .

### PR DESCRIPTION
**Pull request by dacacioa**
New function normalize_audio to improve audio normalization.

### CODE REVIEW

File: audio.py [hunk 1]
```diff
@@ -52,3 +52,9 @@ def glue_tracks(tracks):
     for audio, fade in tracks[1:]:
         final = final.append(audio, crossfade=fade)
     return final
+
+def normalize_audio(podcast_file):
+    """
+    This function normalize track
+    """
+    return normalize(podcast_file, headroom=-1.5)
```
Comment (edubxb) [inline audio.py:60]: What's the reason to use `-1.5` for the `headroom` parameter? _trial & error_?

Comment (dacacioa) [reply audio.py:60]: I did a lot of test and -1.5 was the best setting to our kind of audio (Audacity set by default -1.0). Yes, trial & error.

File: audio.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import re
 import requests
 from pydub import AudioSegment
-
```
Comment (ifosch) [inline audio.py:4]: This line should not be removed.

File: audio.py [hunk 3]
```diff
@@ -52,3 +52,9 @@ def glue_tracks(tracks):
     for audio, fade in tracks[1:]:
         final = final.append(audio, crossfade=fade)
     return final
+
```
Comment (ifosch) [inline audio.py:55]: Extra blank line should be needed here, IIRC.

**Pull request merged**